test/Streamly/Test/Data/Fold.hs

```diff
@@ -518,9 +660,12 @@ main = hspec $ do
 
         -- Partitioning
         prop "partition" Main.partition
+        prop "partitionBy" partitionBy
```

Comment:
Not required. We are testing for `partitionByM`

Reply:
Please check if `partitionBy` is represented in terms of `partitionByM` before removing this.